commands/command_clean.go

```diff
@@ -44,6 +44,10 @@ func cleanCommand(cmd *cobra.Command, args []string) {
 		file.Close()
 	}
 
+	if err != nil {
+		Panic(err, "Error cleaning asset.")
```

Comment:
This causes a test to fail and I could need some advice on how to solve. Should we really ignore cleaning failures?
Do I understand the purpose of the test TestCleanPointer correctly that it verifies that we should do nothing when cleaning a file with content matching a cleaned pointer? Do we need this check? If so should we perhaps add that check explicitly and not bypass all clean errors?